app/views/documents/_form.html.erb

```diff
@@ -1,12 +1,12 @@
 <% collection ||= @collection || locals[:collection] %>
-<%= form_with url: collection_documents_path(collection), html: { id: "document_form", class: "mt-3 w-full" } do |f| %>
+<%= form_with url: collection_documents_path(collection), html: { id: "document_form", class: "mt-3 w-full"} do |f| %>
   <div class="flex justify-center">
     <div class="w-full shadow-lg bg-secondary-200 dark:bg-secondary-900 dark:border-secondary-400 border px-6 py-5 mt-3 rounded-lg">
-        <div class="mb-2 text-sm">Add document</div>
+        <div class="mb-2 text-sm">Add document or URL</div>
         <div class="flex-col">
 
           <%= f.file_field :file, class: "w-full p-2 rounded-lg my-2 px-5 py-2 border border-secondary-400 dark:border-secondary-600" %>
-
+          <%= f.url_field "link", class: "w-full p-2 rounded-lg my-2 px-5 py-2 border border-secondary-400 dark:border-secondary-600" %>
```

Comment:
This looks good in light mode, but in dark mode the text is light but the bg is white.

Can you theme it like the search bar on the same page?

Reply:
Also, it would be good if there was some hint for the user as to what the link field was. It does say "Add document or URL" above the file upload box, but maybe if you added placeholder text it would help?

Reply:
This looks good in dark mode:

```suggestion
          <%= f.url_field "link",
            placeholder: "Enter a URL",
            class: "w-full p-2 rounded-lg my-2 px-5 py-2 border border-secondary-400 dark:bg-secondary-925 dark:border-secondary-600"
          %>
```